### PR TITLE
Fix syntax in examples

### PR DESCRIPTION
**Pull request by Fexiven**
## Community Contribution License
All community contributions in this pull request are licensed to the project maintainers
under the terms of the [Apache 2 license](https://www.apache.org/licenses/LICENSE-2.0). 
By creating this pull request I represent that I have the right to license the 
contributions to the project maintainers under the Apache 2 license.

## Description
There was a minor synax mistake where the custom json and the bucket location was swapped

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Optimization (provides speedup with no functional changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Unit tests added/updated
- [x] Internal documentation updated
- [ ] Create a documentation update request [here](https://github.com/minio/docs/issues/new?label=doc-change,title=Doc+Updated+Needed+For+PR+github.com%2fminio%2fmc%2fpull%2fNNNNN)

### CODE REVIEW

**Comment by Fexiven** [comment]
https://github.com/minio/mc/issues/4761 the USAGE section was updated, the examples not

File: cmd/anonymous-main.go [hunk 1]
```diff
@@ -79,8 +79,8 @@ EXAMPLES:
   4. Set anonymous to "public" for bucket with prefix on Amazon S3 cloud storage.
      {{.Prompt}} {{.HelpName}} set public s3/public-commons/images
 
-  5. Set a custom prefix based bucket anonymous on Amazon S3 cloud storage using a JSON file.
-     {{.Prompt}} {{.HelpName}} set-json s3/public-commons/images /path/to/anonymous.json
+  5. Set bucket to "custom" on Amazon S3 cloud storage using a JSON file.
```
Comment (shtripat) [inline cmd/anonymous-main.go:82]: I think old description looks fine. Just uses command should change.

Comment (Fexiven) [reply cmd/anonymous-main.go]: I reverted to the old description and just swapped the command, also in the comment on the bottom of the file

**Review by shtripat — APPROVED** [review]
lgtm

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**